Packs/OpenPhish/ReleaseNotes/2_0_1.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### New: OpenPhish v2
+**Breaking Change** url command changed to return multiple entries (entry per indicator) instead of a single entry.
```

Comment:
Just make sure to update the docs (if there are any) to show the change.

Reply:
There is no way to display this change via the readme - it should mainly affect (if at all) custom scripts using `executeCommand`